### ISSUE TITLE
Create an installer for Free Manga Downloader

### ISSUE THREAD

**Issue by anonsubmitter** [riderkick/FMD#143]
Can you create an installer for Free Manga Downloader using [Inno Setup](http://jrsoftware.org/isinfo.php) or [NSIS](http://nsis.sourceforge.net/Main_Page)? Having a simple installer would be so much convenient than having to build the software from the source code.

[GitHub can host installers](https://help.github.com/articles/creating-releases/) and [FossHub](http://www.fosshub.com/) and [Toolslib](http://toolslib.net/) are great sites for hosting installers as well.

**Comment by zackad** [comment]
I don't think there's the need to create installer since some people (like me) prefer using portable software. In case you don't know how to compile the code, just download compiled version from [Releases](https://github.com/riderkick/FMD/releases) page.

**Comment by anonsubmitter** [comment]
Oh, I did not see that. I assumed that I would have to compile the whole thing from the source code to use it since it didn't mention any portable version on the main page.

In that case, nevermind. I also like portable versions since they work on computers where the user doesn't have admin access and they don't clog up the registry.

I think that this issue was resolved before I even wrote it. I should have looked more thoroughly for the downloads.
